api_spec.yaml

```diff
@@ -5134,16 +5150,95 @@ paths:
                     type: boolean
         '400':
           $ref: '#/components/responses/400'
-        
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/connectionuridomain/v2:
+    put:
+      tags:
+        - Multitenancy Recipe
+
+      description: |
+        Create or update a connection uri domain.
+
+        SuperTokens subscription license key is required, if not using the base connection uri domain.
+
+        This request must originate from public app and public tenant on the base
+        connection uri domain.
+
+        This `firstFactors` can be set to following values:
+
+        - `null`: When set to `null`, the SDK will use firstFactors defined in the SDK
+        - `[]` (empty array): No first factors would be enabled for the tenant
+        - non-empty array: The first factors that are enabled for the tenant
```

Comment:
These three points are very unclear.

Reply:
For example when you say "the SDK will use" - what SDK? This is just an API spec. Another example, is `non-empty array: The first factors that are enabled for the tenant` -> when reading it, it makes no sense.